creditcoin-js/src/examples/loan-cycle.ts

```diff
@@ -227,4 +227,6 @@ export const main = async () => {
     await api.disconnect().catch(console.error);
 };
 
-main().catch(console.error);
+if (require.main === module) {
```

Comment:
I think we should rename this to something like `fullLoanCycleExample` because importing `main` is kind of weird.